sdcm/cluster_aws.py

```diff
@@ -578,6 +570,16 @@ def _wait_public_ip(self):
             raise PublicIpNotReady(self._instance)
         LOGGER.debug("[{0._instance}] Got public ip: {0._instance.public_ip_address}".format(self))
 
+    def config_ipv6_as_persistence(self):
+        persistence_ipv6_script = dedent("""
+                        BASE_EC2_NETWORK_URL=http://169.254.169.254/latest/meta-data/network/interfaces/macs/
+                        MAC=`curl -s ${BASE_EC2_NETWORK_URL}`
+                        IPv6_CIDR=`curl -s ${BASE_EC2_NETWORK_URL}${MAC}/subnet-ipv6-cidr-blocks`
+                        sudo sh -c  "echo 'sudo ip r a $IPv6_CIDR dev eth0' >>  /etc/sysconfig/network-scripts/init.ipv6-global"
+                        sudo systemctl restart network
```

Comment:
Not needed to restart network.
The first time you set the command in the file, it should boot with routing configured.